integration

```diff
@@ -1475,6 +1475,7 @@
   "youdroid/home-assistant-gitea",
   "youdroid/home-assistant-gogs",
   "youdroid/home-assistant-sickchill",
+  "Youtilitics/home-assisstant",
```

Comment:
The repository is private or non-existent.
Anyway, it can not be used with HACS.
<https://github.com/Youtilitics/home-assisstant>

Reply:
It can be used with HACS (I use it), the URL is from the organization repository. 

I thought you were not compatible with organization repositories for the PR only, do you mean the actual code for the integration can't be hosted by an organization?

Reply:
PS what is the link you shared? It has an extra `s`.

Reply:
<img width="612" alt="image" src="https://github.com/user-attachments/assets/cbf84b20-98ce-4a84-b3b8-2422084f3dcd" />